ui/components/component-library/avatar-base/avatar-base.test.tsx

```diff
@@ -55,12 +55,12 @@ describe('AvatarBase', () => {
   });
   // children
   it('should render children', () => {
-    render(
+    const { container } = render(
       <AvatarBase data-testid="avatar-base">
         <img width="100%" src="./images/arbitrum.svg" />
       </AvatarBase>,
     );
-    const image = screen.getByRole('img');
+    const image = container.querySelector('img');
```

Comment:
Ah because we are updating the wrapping div to `role="img"`  we now have to get the selector rather use role. Makes sense.

Reply:
Ah so I compared our html after I wrote this. I think the above would prevent the need to update the tests as well.

Reply:
What's the goal here? If we update so that it handles the above situation, then it will re-read the image

Reply:
Do we want the Avatar to be read at all times except for the fallback letter? 


https://github.com/MetaMask/metamask-extension/assets/26469696/ba430193-eb71-4fbc-89b3-e63961324a1f